src/i18n/locales/en/translation.json

```diff
@@ -291,7 +291,7 @@
         },
         "login": {
           "title": "Connect to your Kollider account",
-          "description": "Don't have an account already? <0>Sign up</0> now!"
+          "description": "Don't have an account already? Sign up now!"
```

Comment:
@im-adithya Can you please update this PR like we discussed last week? This messages makes no sense here.

Reply:
Fixed it to have links! Ready to merge now!